frontend/src/App.js

```diff
@@ -39,8 +40,11 @@ function App() {
          <Header />
          <Switch>
             <Route path={"/login"}>
-               <LoginPage />
+               <LoginPage/>
```

Comment:
```suggestion
               <LoginPage />
```
I think the space is actually supposed to be there in a self closing tag